lib/_http_client.js

```diff
@@ -283,7 +274,7 @@ ClientRequest.prototype._implicitHeader = function _implicitHeader() {
 
 ClientRequest.prototype.abort = function abort() {
   if (!this.aborted) {
-    process.nextTick(emitAbortNT, this);
+    process.nextTick(emitAbortNT.bind(this));
```

Comment:
If we use `.bind()` in any of these cleanup PRs, we won't be able to/shouldn't backport to v6.x or v4.x.